blocks/loader.ts

```diff
@@ -174,15 +174,23 @@ const wrapLoader = (
       const start = performance.now();
       let status: "bypass" | "miss" | "stale" | "hit" | undefined;
       const cacheKeyValue = cacheKey(props, req, ctx);
+      const bypassCache = mode === "no-store" ||
+        !isCache(maybeCache) ||
+        cacheKeyValue === null;
```

Comment:
Can you split this line into different `const` and give them a explaining name?

Reply:
Sure, fixed